wca-regulations.md

```diff
@@ -76,7 +76,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 2d) A competitor's name, country, gender, and competition results are considered public information. All other personal information is considered confidential, and must not be disclosed to outside organizations/persons without the consent of the competitor.
 - 2e) Competitors must represent a country of which they hold citizenship. The WCA Delegate should verify citizenship standing by means of documents (e.g. a passport) at their first competition. If a competitor is found ineligible to represent the country under which they have registered, the competitor may be disqualified retroactively and/or suspended, at the discretion of the WCA Board.
     - 2e1) The eligible countries are defined by the [WCA List of Recognized Countries](https://www.worldcubeassociation.org/regulations/countries/).
-    - 2e2) Competitors with updates to their citizenship status may change their country of representation in their first competition of a calendar year.
+    - 2e2) Competitors with updates to their citizenship status may change their country of representation in their first competition of a calendar year. Exception: if the citizenship of the country which the competitor represents is lost, they must change the country of representation at their next competition.
```

Comment:
We first say "in their first competition" and then "at their next competition", which preposition is correct in this context?